Ports/flare-game/package.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=flare-game
```

Comment:
Instead of having two ports, can you make a single port called `flare` that downloads & installs both the engine and the game files? Or is there a good reason to keep them separated?

Reply:
Flare is an engine for which campaigns are made (kind of like ScummVM), the game port is just the example campaign made by the engine devs

Reply:
I considered mentioning it in the list of available ports, but I didn't find a good, short description of it...